pages/country.js

```diff
@@ -45,8 +46,19 @@ const AnimatedHeading = styled(Heading)`
 `
 
 export default class Country extends React.Component {
-  static async getInitialProps ({ query }) {
+  static async getInitialProps ({ req, res, query } ) {
```

Comment:
Needs a little reformatting. Remove the extra space toward the end `...query })`